packages/tools/src/tools/annotation/PlanarFreehandContourSegmentationTool.ts

```diff
@@ -10,6 +10,7 @@ class PlanarFreehandContourSegmentationTool extends PlanarFreehandROITool {
       {
         configuration: {
           calculateStats: false,
+          allowOpenContours: false,
         },
```

Comment:
still not sure what is allowOpenContours compared to closed status

Reply:
Add docs there.